SUP_Board_JSON.txt

```diff
@@ -0,0 +1,9 @@
+{
+  "brand": "Decathlon",
+  "model": "Hurley",
+  "size": "335X85X15",
+  "color": "Mixed Color",
+  "maximum load": "150kg",
+  "weight of the packaging" : "13kg",
+  "price" : "299euro"
```

Comment:
Same goes here, when we're able to use numeric value (like integer) -> better we use it

```json
"price_eur": 299
```